src/server/pps/server/api_server.go

```diff
@@ -3855,3 +3856,26 @@ func (a *apiServer) SetClusterDefaults(ctx context.Context, req *pps.SetClusterD
 	}
 	return &resp, nil
 }
+
+func (a *apiServer) GetProjectDefaults(ctx context.Context, req *pps.GetProjectDefaultsRequest) (*pps.GetProjectDefaultsResponse, error) {
+	var projectDefaults ppsdb.ProjectDefaultsWrapper
+	if req.Project == nil {
```

Comment:
I thought we had some utility method that does this basic check on Project structs

Reply:
There’s `EnsureProject` on repos, but that sets the project if it's missing. Since this is a new API, I think it makes sense to require that the project be explicit.